src/palindrom-js-listener.js

```diff
@@ -1,4 +1,5 @@
 (function(global) {
+  const historyLength = Number(localStorage['starcounter-debug-aid-patch-history-length'] || 100);
```

Comment:
Other pages use `getCurrentWindow().localStorage` but the above line does not. Please clarify.